sandbox/src/mocks/bookOrDeferAppointment/requests/DeferralWithSlotReference.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "appointment",
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Appointment-1",
+      "https://fhir.hl7.org.uk/STU3/StructureDefinition/CareConnect-Appointment-1"
+    ]
+  },
+  "language": "en",
+  "resourceType": "Appointment",
+  "status": "proposed",
+  "reason": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReviewReason-1",
+          "code": "SLOT_NOT_AVAILABLE"
```

Comment:
I think this should be using the "BOOKING_ATTEMPT_PROBLEM" reason when including a slot. If I remember correctly the "SLOT_NOT_AVAILABLE" reason is when attempting to book to a service that doesn't have any slots available.

Reply:
Looking at this again this looks to be correct as it is the "SLOT_NOT_AVAILALBLE" where the client is expected to provide the details of the slot they attempted to book to.